CHANGELOG.md

```diff
@@ -4938,3 +4939,4 @@
 [@em-gazelle]: https://github.com/em-gazelle
 [@tleish]: https://github.com/tleish
 [@pbernays]: https://github.com/pbernays
+[@rdunlop]: https://github.com/rdunlop
```

Comment:
#8802 has been merged. Can you remove it and rebase with the latest master branch?